runtime/src/accounts.rs

```diff
@@ -1231,8 +1233,12 @@ impl Accounts {
         durable_nonce: &DurableNonce,
         lamports_per_signature: u64,
         leave_nonce_on_success: bool,
-    ) -> Vec<(&'a Pubkey, &'a AccountSharedData)> {
+    ) -> (
+        Vec<(&'a Pubkey, &'a AccountSharedData)>,
```

Comment:
I was tried this solution at the first version of implementation that was not submitted for review. This approach causes more complex changes in source code because it requires the same tuple to be introduced into more general functions like  AccountsDb::store or to make re-build of collections every time when signatures are not meant to be provided. Also it will lead to more changes in tests. So I decided to keep this separate vectors.

Reply:
sounds good